test/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Microsoft/hcsshim/test
 go 1.13
```

Comment:
You haven't added anything to the test/ directory, should this change be here?

Reply:
Not sure why it loves to update test deps when saving sometimes, but it's also harmless so *shrugs*. I can remove both